libs/providers/vmware.py

```
@@ -1268,6 +1268,20 @@ def clone_vm(
+                disk_locators: list[vim.vm.RelocateSpec.DiskLocator] = []
+                for device in source_vm.config.hardware.device:
+                    if isinstance(device, vim.vm.device.VirtualDisk):
+                        locator = vim.vm.RelocateSpec.DiskLocator()
+                        locator.diskId = device.key
+                        locator.datastore = target_datastore or device.backing.datastore
+                        locator.diskBackingInfo = vim.vm.device.VirtualDisk.FlatVer2BackingInfo()
+                        locator.diskBackingInfo.eagerlyScrub = True
+                        locator.diskBackingInfo.thinProvisioned = False
+                        disk_locators.append(locator)
+                relocate_spec.disk = disk_locators
+                LOGGER.info(f"Set eagerlyScrub=True on {len(disk_locators)} disk(s) for thick-eager provisioning.")
```

Ruff (0.15.11)

[warning] Logging statement uses f-string

(G004)